### PR TITLE
feat(windows): add right modifier included in hotkey optional functionality

### PR DESCRIPTION
**Pull request by rc-swag**
Step 2. of Fixes:#11471

# User Testing

## TEST_LANGUAGE_HOTKEYS_LEFT_SIDE

For this test only the left and right modifier keys will trigger the hotkeys
1. Install Keyman
2. Install some keyboard layouts Tamil and IPA SIL
3. Open Configuration and assign hotkeys to <kbd>Ctrl</kbd>+<kbd>G</kbd> to Tamil and 
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>to IPA SIL
5. Open Notepad and type some text 
6. Switch to Tamil using the Hotkey press <kbd>Left Ctrl</kbd>+<kbd>G</kbd>
7. Verify that it has switched in the task and language bar to Tamil and type some text.
8. Switch to IPA SIL using the Hotkey press <kbd>Left Ctrl</kbd>+<kbd>Left Shift</kbd>+<kbd>I</kbd>
9. Verify it has it has switched in the task and language bar to IPA SIL and type some text.

## TEST_LANGUAGE_HOTKEYS_RIGHT_SIDE

For this test only the left and right modifier keys will trigger the hotkeys
1. Install Keyman
2. Install some keyboard layouts Tamil and IPA SIL
3. Open Configuration and assign hotkeys to <kbd>Ctrl</kbd>+<kbd>G</kbd> to Tamil and 
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>to IPA SIL
4. In Configuration Options tab click the check box for "Use right modifier for hotkeys"
5. Open Notepad and type some text 
6. Switch to Tamil using the Hotkey  press <kbd>Right Ctrl</kbd>+<kbd>G</kbd>
7. Verify that it has switched in the task and language bar to Tamil and type some text.
8. Switch to IPA SIL using the Hotkey  press <kbd>Ctrl</kbd>+<kbd>Right Shift</kbd>+<kbd>I</kbd>
9. Verify it has it has switched in the task and language bar to IPA SIL and type some text.

### CODE REVIEW

**Comment by dinakaranr** [comment]
# Test Results
I tested this issue with the attached "Keyman 18.0.95-alpha-test-12259" build on the Windows 10 OS. Here is my observation.

* **TEST_LANGUAGE_HOTKEYS_LEFT_SIDE (Passed):**
1. Installed the "Keyman-18.0.95.apk" file and gave all permissions to the application.
2. Open the Keyman configuration dialog by pressing the icon from Windows tray.
3. Installed the Tamil99 and IPA SIL keyboards.
4. Navigate to the "HotKeys" tab. Here, I assigned the left side of CTRL+G to Tamil and CTRL_Shift+I to IPA SIL.
5. Open the notepad. Press the left side of CTRL+G and then Enter some letters.
6. Here, Verified that the Tamil letters appeared
7. Press the Left side of the Ctrl+Shift+I key and then Enter some letters.
Here, Verified that the keyboard switched from Tamil to IPA keyboard using the hotkeys.
Verified that the IPA letters appeared.
It works well.

* **TEST_LANGUAGE_HOTKEYS_RIGHT_SIDE (Passed):**
1. Installed the "Keyman-18.0.95.apk" file and gave all permissions to the application.
2. Open the Keyman configuration dialog by pressing the icon from the Windows tray.
3. Installed the Tamil99 and IPA SIL keyboards.
4. Navigate to the "HotKeys" tab. Here, I assigned the right side of CTRL+G to Tamil and CTRL_Shift+I to IPA SIL.
5. Open the notepad. Press the right side of CTRL+G and then Enter some letters.
6. Here, Verified that the Tamil letters appeared.
7. Press the right side of the Ctrl+Shift+I key and then Enter some letters.
8. Here, Verified that the keyboard switched from Tamil to IPA keyboard using the hotkeys.
Verified that the IPA letters appeared.
It works well. Thank you.
Please refer to the screenshot below

File: common/windows/cpp/include/registry.h [hunk 1]
```diff
@@ -110,7 +110,7 @@
 #define REGSZ_KeyboardHotkeysAreToggle "hotkeys are toggles"
 #define REGSZ_DeadkeyConversionMode    "deadkey conversion mode"                // CU   // I4552
 #define REGSZ_ZapVirtualKeyCode        "zap virtual key code"   // LM, defaults to 0x0E (_VK_PREFIX_DEFAULT)
-/* Non-chiral use of hotkeys instead of left-only hotkeys */
+/* Default is to only use left modifier in hotkeys trigger */
```
Comment (ermshiperete) [inline common/windows/cpp/include/registry.h:113]: :+1: This comment is much clearer now

**Review by ermshiperete — APPROVED** [review]
LGTM

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.106-alpha](https://keyman.com/downloads/releases/alpha/18.0.106)

**Pull request merged**